exporter/signalfxexporter/translation/translator.go

```diff
@@ -185,6 +185,10 @@ type Rule struct {
 	// excluded by aggregation.
 	WithoutDimensions []string `mapstructure:"without_dimensions"`
 
+	// WithDimensions used by "rename_metrics" translation rule to add dimensions that are necessary for
+	// existing SFx content for desired metric name
+	WithDimensions map[string]string `mapstructure:"with_dimensions"`
```

Comment:
I wonder if this should be called `AddDimensions`, because `WithDimensions` sort of implies that we're only renaming metrics that already have these dimensions, but what we're actually doing is adding these dimensions.

Reply:
Good call, will update.